coreos-installer

```diff
@@ -662,31 +682,135 @@ log() {
     echo "$1" >&2
 }
 
+#########################################################
+#Update bootloader if needed
+#########################################################
+write_bootloader() {
```

Comment:
nit: `write_bootloader` is, at least currently, `s390x` specific. This may be a little confusing since boot loader functionality may not always come from this function (EG: see `x86_64`). I suggest either renaming the function. Possibly something like `write_special_bootloader` or `write_arch_dependent_bootloader`, etc..  or something much better than those :smile: Just a suggestion.

Reply:
I made it to `write_zipl_bootloader` and update the comment also.